ksql-execution/src/main/java/io/confluent/ksql/execution/codegen/CodeGenSpec.java

```
@@ -77,14 +83,20 @@ public void resolve(GenericRow row, Object[] parameters) {
     }
   }
 
+  public String getSchemaName(StructExpression structExpression) {
+    return structToCodeName.get(structExpression);
```

throw exception if not found, rather than returning `null`.